src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-user-listing/sw-users-permissions-user-listing.html.twig

```diff
@@ -203,6 +186,15 @@
         </template>
     </sw-data-grid>
     {% endblock %}
+
+    <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
+    {% block sw_settings_user_list_content_confirm_password_modal %}
```

Comment:
can you please add testcases that the workflow with the new modal component works?

Reply:
I‘ll add some. Originally I didn’t add some, as the old modal wasn’t covered by tests, too.